### PR TITLE
Chore(release): Prepare for v3.9.1 release

### PR DESCRIPTION
**Pull request by The-inside-man**
Summary
-------

-  Update Changelog to reflect Buxfix
-  Update version to 3.9.1

Preparing for release v3.9.1

Test plan
---------
-  FSC Suite

Issues
------

Bugfix - Fixed issue with serving incorrect variation in projects containing multiple flags with duplicate keys

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
 # Optimizely Python SDK Changelog
 
+## 3.9.1
+July 14th, 2021
+
+### Bug Fixes:
+* Fixed issue with serving incorrect variation in projects containing multiple flags with duplicate keys. [#347] (https://github.com/optimizely/python-sdk/pull/347)
```
Comment (msohailhussain) [inline CHANGELOG.md:7]: add one more PR which has event_dispatcher fix.

Comment (The-inside-man) [reply CHANGELOG.md:7]: updated.

**Review by msohailhussain — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+* Fixed issue with serving incorrect variation in create_impression_event in user_event_factory.py. [#350] (https://github.com/optimizely/python-sdk/pull/350)
+
 ## 3.9.0
 June 1st, 2021
 
 ### New Features
 * Added support for multiple concurrent prioritized experiments per flag. [#322](https://github.com/optimizely/python-sdk/pull/322)
 
-
```
Comment (keppel2) [inline CHANGELOG.md:9]: Not clear why this is removed.

Comment (The-inside-man) [reply CHANGELOG.md:9]: There were too many spaces between comments, it did not match others

Comment (keppel2) [reply CHANGELOG.md:9]: That can be done on a separate formatting PR.

**Pull request merged**